### ISSUE TITLE
Can not transform components defined with `VariableDeclaration`

### ISSUE THREAD

**Issue by SukkaW** [lxsmnsyc/forgetti#10]
**Input**

```jsx
export const Example = () => {
  const value = [1, 2, 3, 4];

  return (
    <div>
      {value.map(i => <p key={i}>{i}</p>)}
    </div>
  );
};
```

**Output**

```js
import { jsx as _jsx } from "react/jsx-runtime";
export const Example = function _Component() {
    const value = [
        1,
        2,
        3,
        4
    ];
    return /*#__PURE__*/ _jsx("div", {
        children: value.map((i)=>/*#__PURE__*/ _jsx("p", {
                children: i
            }, i))
    });
};
```

Forgetti simply doesn't transform the component here.

**Comment by lxsmnsyc** [comment]
Hmmm, weird, I thought I tested this

**Comment by lxsmnsyc** [comment]
Should be fixed in 0.4.5

**Comment by SukkaW** [comment]
> Should be fixed in 0.4.5

I can confirm that 0.4.5 has fixed the issue! Great jobs!
